apdx-validation.tex

```diff
@@ -78,88 +83,243 @@ \subsubsection*{Rules for SBOL Objects}
+\sbol{ComponentReference} &\sbol{Feature} & \sbol{inChildOf}, \sbol{hasFeature} & \sec{sec:ComponentReference} \\
+\sbol{Component} 	& \sbol{TopLevel} 	& \sbolmult{type:C}{type}, \sbolmult{role:C}{role}, \sbolmult{hasSequence:C}{hasSequence}, \sbol{hasFeature}, \sbol{hasInteraction},  \sbol{hasConstraint}, \sbol{hasModel}, \sbol{hasInterface} & \sec{sec:Component} \\
+\sbol{Constraint} 	& \sbol{Identified} 	& \sbol{subject}, \sbol{object}, \sbol{restriction} & \sec{sec:Constraint} \\
+\sbol{Cut} 			& \sbol{Location} 	& \sbol{at} & \sec{sec:Cut} \\
+\sbol{EntireSequence} & \sbol{Location} 	& 	& \sec{sec:EntireSequence} \\
+\sbol{ExperimentalData} & \sbol{TopLevel} &	& \sec{sec:ExperimentalData} \\
+\sbol{Experiment} 	& \sbol{Collection} 	&	& \sec{sec:Experiment} \\
+\sbol{ExternallyDefined} & \sbol{Feature}	& \sbolmult{type:ED}{type}, \sbolmult{definition:ED}{definition} & \sec{sec:ExternallyDefined} \\
+\sbol{Feature} 		& \sbol{Identified} 	& \sbolmult{role:F}{role}, \sbolmult{orientation:F}{orientation} & \sec{sec:Feature} \\
+\sbol{Identified} 	& none 			& \sbol{displayId}, \sbol{name}, \sbol{description}, \sbol{hasMeasure} & \sec{sec:Identified} \\
+\sbol{Implementation} & \sbol{TopLevel} 	& \sbol{built} & \sec{sec:Implementation} \\
+\sbol{Interaction} 	& \sbol{Identified} 	& \sbolmult{type:I}{type}, \sbol{hasParticipation} & \sec{sec:Interaction} \\
+\sbol{Interface} 		& \sbol{Identified} 	& \sbol{input}, \sbol{output}, \sbol{nondirectional} & \sec{sec:Interface} \\
+\sbol{LocalSubComponent} & \sbol{Feature} & \sbolmult{type:LSC}{type}, \sbolmult{hasLocation:LSC}{hasLocation} & \sec{sec:LocalSubComponent} \\
+\sbol{Location}		& \sbol{Identified} 	& \sbolmult{orientation:L}{orientation}, \sbol{order}, \sbolmult{hasSequence:L}{hasSequence} & \sec{sec:Location} \\
+\sbol{Model} 		& \sbol{TopLevel} 	& \sbolmult{source:M}{source}, \sbol{language}, \sbol{framework} & \sec{sec:Model} \\
+\sbol{Participation} 	& \sbol{Identified} 	& \sbolmult{role:P}{role}, \sbol{participant}, \sbol{higherOrderParticipant} & \sec{sec:Participation} \\
+\sbol{Range} 		& \sbol{Location} 	& \sbol{start}, \sbol{end} & \sec{sec:Range} \\
+\sbol{SequenceFeature} & \sbol{Feature}	& \sbolmult{hasLocation:SF}{hasLocation} & \sec{sec:SequenceFeature} \\
+\sbol{Sequence} 	& \sbol{TopLevel} 	& \sbol{elements}, \sbol{encoding} & \sec{sec:Sequence} \\
+\sbol{SubComponent} & \sbol{Feature} 	& \sbol{roleIntegration}, \sbol{instanceOf}, \sbol{sourceLocation}, \sbolmult{hasLocation:SC}{hasLocation} & \sec{sec:SubComponent} \\
+\sbol{TopLevel}		& \sbol{Identified} 	& \sbol{hasNamespace}, \sbol{hasAttachment} & \sec{sec:TopLevel} \\
+\sbol{VariableFeature} & \sbol{Identified} 	& \sbol{cardinality}, \sbol{variable}, \sbol{variant}, \sbol{variantCollection}, \sbol{variantDerivation}, \sbol{variantMeasure} & \sec{sec:VariableFeature} \\
+\hline
+\prov{Activity}		& \sbol{TopLevel}	&	& \sec{sec:prov:Activity}\\
```

Comment:
I believe `sbol:type` is now a property of `Activity` that belongs in this column of the table

Reply:
Good catch; it was already in Table 22 as well.  I've added it now.